corehq/motech/repeaters/models.py

```diff
@@ -175,16 +176,17 @@ class Repeater(QuickCachedDocumentMixin, Document):
 
     domain = StringProperty()
 
-    # TODO: (2020-03-06) Migrate to ConnectionSettings
+    connection_settings_id = IntegerProperty(required=False, default=None)
```

Comment:
Thoughts on making this a foreign key reference instead?  Seems like it might be a bit messy to do the automatic creation business, but from a data modeling perspective, I imagine that'd be nicer.

[`default` can be a callable](https://docs.djangoproject.com/en/2.2/ref/models/fields/#default), so you may be able to use that to create these automatically.

Reply:
> Thoughts on making this a foreign key reference instead?

Can I do that while Repeaters are in Couch? (I would love to get Repeaters into Postgres (and refactor the way their forms and views work).)

> `default` can be a callable, so you may be able to use that to create these automatically.

Ooo! I didn't know that. That might be very cool.

Reply:
> Can I do that while Repeaters are in Couch? (I would love to get Repeaters into Postgres (and refactor the way their forms and views work).)

Oh right - must've glossed over that aspect of this.  Carry on.